src/util/mapbox.js

```diff
@@ -448,8 +447,8 @@ export class TurnstileEvent extends TelemetryEvent {
             this.fetchEventData();
         }
 
-        const tokenData = parseAccessToken(config.ACCESS_TOKEN);
-        const tokenU = tokenData ? tokenData['u'] : config.ACCESS_TOKEN;
+        const tokenData = parseAccessToken(RequestManager._customAccessToken || config.ACCESS_TOKEN);
```

Comment:
This isn't going to work. `_customAccessToken` isn't a static property; it's set in the constructor function so it will only be available on instances of `RequestManager` and `RequestManager._customAccessToken` will always be `undefined`. Why not use the `customAccessToken` parameter being passed into the function? Or even create `this._customAccessToken` in the `TurnstileEvent` constructor and set it in `postTurnstileEvent` so you can reference it in `processRequest` instead of passing the variable down the function chain? That's what I did with the `MapLoad` event needing `skuToken`.